### ISSUE TITLE
Add code of conduct file

### ISSUE THREAD

**Issue by Kerveyfelix** [PragatiVerma18/Snappify#15]
All open-source projects have code of conduct file. Can I make this file?

**Comment by yashgautam9274** [comment]
I want to work on this issue

**Comment by PragatiVerma18** [comment]
@yashgautam9274 go ahead. Thanks for suggesting!

**Comment by Kerveyfelix** [comment]
@PragatiVerma18 I opened this issue. So can I work on this?

**Comment by yashgautam9274** [comment]
@PragatiVerma18 I sended the PR. Review it.

**Comment by yashgautam9274** [comment]
@Kerveyfelix Yeah you opened this issue. Actually In all project You opened this type of issue and You are at top position.
Congrulation!

**Comment by Kerveyfelix** [comment]
@s-ayush2903 Kindly look into this matter. I didn’t expect this kind of behaviour in NWoC 😢.

**Comment by Kerveyfelix** [comment]
> @Kerveyfelix Yeah you opened this issue. Actually In all project You opened this type of issue and You are at top position.
> Congrulation!

@s-ayush2903 kindly look into this matter. I didn’t expect such rude behaviour of people in NWOC.

**Comment by Kerveyfelix** [comment]
Congratulations to @yashgautam9274 for all the points getting from updating README.MD files only. Unlike you I have a balance of coding as well as documentation experience.
